### DISCUSSION TITLE
월드 제작시 참고할 가이드라인 목록입니다.

### DISCUSSION THREAD

**Discussion by Zepeto-TechEvangelist** [naverz/zepeto-studio-kor#595] [1. Announcements]
안녕하세요. 월드 제작시 참고할 가이드라인 목록입니다.
컨텐츠 기획 단계에서 충분히 숙지하여 주시기를 부탁 드립니다. 
감사합니다. 

Hello, this is a list of guidelines for world contents.
We ask that you be fully aware of this during the content planning stage.
Thank you.

<hr/>

1. [제페토 스튜디오 창작물 공통 가이드라인](https://studio.zepeto.me/kr/guides/common-guidelines)

2. [제페토 월드 가이드라인 ](https://studio.zepeto.me/guides/world-guidelines)

3. [제페토 커뮤니티 가이드라인](https://support.zepeto.me/hc/ko/articles/900005448466-%EC%BB%A4%EB%AE%A4%EB%8B%88%ED%8B%B0-%EA%B0%80%EC%9D%B4%EB%93%9C%EB%9D%BC%EC%9D%B8)

4. [ZEPETO 활용 콘텐츠(UGC) 이용 가이드라인](https://support.zepeto.me/hc/ko/articles/4414701429145-ZEPETO-%ED%99%9C%EC%9A%A9-%EC%BD%98%ED%85%90%EC%B8%A0-UGC-%EC%9D%B4%EC%9A%A9-%EA%B0%80%EC%9D%B4%EB%93%9C%EB%9D%BC%EC%9D%B8)

**Comment by smilejsu82** [comment]
잘읽어 봐야 겠네요 :)
